From 3818a6b073c83ecc3cba0d2568b687defcc74908 Mon Sep 17 00:00:00 2001
From: Jan Kratochvil <jan.kratochvil@redhat.com>
Date: Tue, 7 Aug 2012 16:49:38 +0000
Subject: [PATCH] gdb/doc/
 	* gdbint.texinfo (Debugging GDB): In section
 	'Debugging @value{GDBN} with itself' change .gdbinit for gdb-gdb.gdb.
 	Mention also gdb-gdb.py.

---
 gdb/doc/ChangeLog      |    6 ++++++
 gdb/doc/gdbint.texinfo |   10 +++++-----
 2 files changed, 11 insertions(+), 5 deletions(-)

Index: gdb-7.2/gdb/doc/gdbint.texinfo
===================================================================
--- gdb-7.2.orig/gdb/doc/gdbint.texinfo
+++ gdb-7.2/gdb/doc/gdbint.texinfo
@@ -7961,11 +7961,11 @@ debugged in another.  Rather than typing
 ./gdb}}, which works on Suns and such, you can copy @file{gdb} to
 @file{gdb2} and then type @kbd{@w{./gdb ./gdb2}}.
 
-When you run @value{GDBN} in the @value{GDBN} source directory, it will read a
-@file{.gdbinit} file that sets up some simple things to make debugging
-gdb easier.  The @code{info} command, when executed without a subcommand
-in a @value{GDBN} being debugged by gdb, will pop you back up to the top level
-gdb.  See @file{.gdbinit} for details.
+When you run @value{GDBN} in the @value{GDBN} source directory, it will read
+@file{gdb-gdb.gdb} file (plus possibly @file{gdb-gdb.py} file) that sets up
+some simple things to make debugging gdb easier.  The @code{info} command, when
+executed without a subcommand in a @value{GDBN} being debugged by gdb, will pop
+you back up to the top level gdb.  See @file{gdb-gdb.gdb} for details.
 
 If you use emacs, you will probably want to do a @code{make TAGS} after
 you configure your distribution; this will put the machine dependent
